keyboards/onepiece/qk21/info.json

```diff
@@ -0,0 +1,62 @@
+{
+    "manufacturer": "linlin012",
+    "keyboard_name": "qk21",
+    "maintainer": "linlin012",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["F7", "E6", "C7", "D5"],
+        "rows": ["F0", "F1", "F4", "F5", "F6", "B4"]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0050",
+        "vid": "0x1EA7"

```

Comment:
This vendor ID is in use, and should be changed.
https://www.the-sz.com/products/usbid/index.php?v=0x1EA7&p=&n=

Specifically, anything above 0x4000 is unassigned, though the further past this, the better. 

Also, you can check against Vendor ID's used in QMK here: https://yanfali.github.io/qmk_usb_usage/